guardian/workflow-frontend
# Add multi-byline to dropdown behind feature switch

## What does this change?

Adds an option to create Multi-byline articles in Workflow. This is behind a feature switch at the moment.

Unfortunately we don't have an icon for multi-byline articles so let's re-use Article one for now.

## How to test

Run locally or deploy to CODE. Look in the "Create new" menu to ensure "Multi-byline" is not present. Hit `shift-f12` and enable the feature switch "Multi-byline". Look in the "Create new" menu to ensure "Multi-byline" is now present. Does the button work correctly?

## Images

![Screenshot 2024-12-02 at 10 53 05](https://github.com/user-attachments/assets/9c911b03-2460-46f8-99c1-c3a63a8af99a)

![Screenshot 2024-12-02 at 11 14 10](https://github.com/user-attachments/assets/4b391665-a16d-45ae-b056-4ba7abe47250)

![Screenshot 2024-12-02 at 14 07 27](https://github.com/user-attachments/assets/5604ae98-94e4-49f2-8479-d078daf98e42)
